liquidsoap.spec

```diff
@@ -43,60 +44,75 @@ BuildRequires: libstdc++-static
 BuildRequires: libvorbis-devel
 BuildRequires: ocaml
 BuildRequires: ocaml-alsa-devel
-BuildRequires: ocaml-biniou
 BuildRequires: ocaml-biniou-devel
-BuildRequires: ocaml-cry
-BuildRequires: ocaml-dtools
-BuildRequires: ocaml-duppy
-BuildRequires: ocaml-easy-format
+BuildRequires: ocaml-cry-devel
+BuildRequires: ocaml-dtools-devel
+BuildRequires: ocaml-duppy-devel
 BuildRequires: ocaml-easy-format-devel
 BuildRequires: ocaml-findlib
-BuildRequires: ocaml-flac
-BuildRequires: ocaml-inotify
+BuildRequires: ocaml-flac-devel
+BuildRequires: ocaml-inotify-devel
 BuildRequires: ocaml-bjack-devel
-BuildRequires: ocaml-ladspa
-BuildRequires: ocaml-lame
-BuildRequires: ocaml-magic
-BuildRequires: ocaml-mm
-BuildRequires: ocaml-opus
-BuildRequires: ocaml-samplerate
-BuildRequires: ocaml-soundtouch
-BuildRequires: ocaml-speex
+BuildRequires: ocaml-ladspa-devel
+BuildRequires: ocaml-lame-devel
+BuildRequires: ocaml-magic-devel
+BuildRequires: ocaml-mm-devel
+BuildRequires: ocaml-ocamldoc
+BuildRequires: ocaml-ogg-devel
+BuildRequires: ocaml-opus-devel
+BuildRequires: ocaml-pcre-devel
+BuildRequires: ocaml-samplerate-devel
+BuildRequires: ocaml-soundtouch-devel
+BuildRequires: ocaml-speex-devel
 BuildRequires: ocaml-ssl-devel
-BuildRequires: ocaml-taglib
+BuildRequires: ocaml-taglib-devel
 BuildRequires: ocaml-theora-devel
-BuildRequires: ocaml-vorbis >= 0.7.0
-BuildRequires: ocaml-xmlm
+BuildRequires: ocaml-vorbis-devel
 BuildRequires: ocaml-xmlm-devel
-BuildRequires: ocaml-xmlplaylist
+BuildRequires: ocaml-xmlplaylist-devel
 BuildRequires: ocaml-yojson-devel
 BuildRequires: opus-devel
-BuildRequires: ocaml-pcre-devel
+BuildRequires: perl-XML-DOM
+BuildRequires: pcre-devel
 BuildRequires: soundtouch-devel
 BuildRequires: speex-devel
 BuildRequires: systemd
```

Comment:
Please add the  `%{?systemd_requires}` macro in order [to include the per-scriptlet deps on systemd](https://fedoraproject.org/wiki/Packaging:Scriptlets#Systemd).

Reply:
Added in 23e5676